src/types/ellipsis.rs

```diff
@@ -15,24 +15,26 @@ impl PyEllipsis {
     }
 }
 
-impl<'v> PyTryFrom<'v> for PyEllipsis {
-    fn try_from<V: Into<&'v PyAny>>(value: V) -> Result<&'v Self, crate::PyDowncastError<'v>> {
-        let value: &PyAny = value.into();
-        if unsafe { ffi::Py_Ellipsis() == value.as_ptr() } {
-            return unsafe { Ok(value.downcast_unchecked()) };
-        }
-        Err(PyDowncastError::new(value, "ellipsis"))
+unsafe impl PyTypeInfo for PyEllipsis {
+    const NAME: &'static str = "ellipsis";
+
+    const MODULE: Option<&'static str> = None;
```

Comment:
`help(type(...))` reports "builtins" as the module. Should we do this as well here or is `None` effectively the same?

Reply:
To be honest `MODULE` is only used in creation of type objects to build the `__qualname__` in pyclass types. I kinda want to deprecate this and move it to `PyClassImpl` as I think `T::type_object(py).getattr("__module__")` is the more correct way to access this if it's needed, as that's guaranteed to be right at runtime.